staging/src/k8s.io/apiserver/pkg/util/flowcontrol/request/width.go

```diff
@@ -34,8 +34,11 @@ const (
 )
 
 type WorkEstimate struct {
```

Comment:
I am becoming less satisfied with the name "WorkEstimate" for this datatype.  For one thing, the word "estimate" is not very helpful here.  It is the _usage_ of this data that determines whether it is estimated or not.  More importantly, this struct does not simply hold "work" but rather components that go into computing work.  The work of a request is now a function of four things (two widths and two durations), three of which are here and one of which is in the queueset (thus far).